src/compile/common.ts

```diff
@@ -130,11 +135,15 @@ export function numberFormat(fieldDef: FieldDef<string>, specifiedFormat: string
 /**
  * Returns the time expression used for axis/legend labels or text mark for a temporal field
  */
-export function timeFormatExpression(field: string, timeUnit: TimeUnit, format: string, shortTimeLabels: boolean, timeFormatConfig: string): string {
+export function timeFormatExpression(field: string, timeUnit: TimeUnit, format: string, shortTimeLabels: boolean, timeFormatConfig: string, isUTC: boolean): string {
   if (!timeUnit || format) {
     // If there is not time unit, or if user explicitly specify format for axis/legend/text.
     const _format = format || timeFormatConfig; // only use config.timeFormat if there is no timeUnit.
-    return `timeFormat(${field}, '${_format}')`;
+    if (isUTC) {
```

Comment:
You handle this for the upper case -- what about the other case (`formatExpression`)?